### DISCUSSION TITLE
questions on template class and  function bind.

### DISCUSSION THREAD

**Discussion by ChunelFeng** [pybind/pybind11#4517] [Q&A]
HI，current we want to bind this cpp repo  [CGraph](https://github.com/ChunelFeng/CGraph)  with python3,

and we want to make sure the limit for template support in pybind11.

As there is a lot of template class and function in CGraph repo,

just ask whether it works ok with pybind11.

**Comment by hzhangxyz** [comment]
Currently, i bind every common used specifications as individual python classes in my project.

**Reply by ChunelFeng** [reply]
i notice you are in China now，can you render me
some help？

my wechat is： ChunelFeng
